services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+from django.db.models import QuerySet
+import datetime
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int,
+                         cinema_hall_id: int
+                         ) -> None:
+    MovieSession.objects.create(show_time=movie_show_time,
+                                movie_id=movie_id,
+                                cinema_hall_id=cinema_hall_id)
+
+
+def get_movies_sessions(session_date: datetime = None) -> QuerySet:
+    movie_session = MovieSession.objects.all()
+    if session_date:
+        movie_session = movie_session.filter(show_time__date=session_date)
+    return movie_session
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(session_id: int,
+                         show_time: datetime = None,
+                         movie_id: datetime = None,
+                         cinema_hall_id: int = None) -> None:

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]